Scripts/Modules/MultipleRayInputModule/Interfaces/IRayClickHandler.cs

```diff
@@ -0,0 +1,14 @@
+﻿#if UNITY_EDITOR
```

Comment:
Should each module have their own interfaces or should this go with the others?

Reply:
I could go either way.  @mschoen what are your thoughts?

Reply:
I'd say we keep at least these with the input module since they're basically overloads of existing Unity UI interfaces.

Also, in general, I'd like to avoid adding to our general list of interfaces, since it's huge and already hard to digest.